docs/developers.rst

```diff
@@ -556,6 +556,90 @@ If everything turned out well, the cloning process should not ask for the creden
    git pull
 
 
+Configuration of Grafana metrics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Existing metrics**
+
+To configure which metrics will be written one should use BEAM configuration files.
+
+There is a configuration entry beam.sim.metric.collector.metrics which contains names of enabled metrics
+and this way controls which metrics will be written during BEAM execution.
```

Comment:
nitpick: Is `this way` here for the same purpose as `thus`?

Reply:
+